oneanddone/users/forms.py

```diff
@@ -20,10 +20,11 @@ class SignUpForm(forms.ModelForm):
         max_length=30, regex=r'^[a-zA-Z0-9]+$',
         error_messages={'invalid': _("This value may contain only alphanumeric characters.")})
     personal_url = MyURLField(label=_('Personal URL:'))
+    bugzilla_email = forms.EmailField(label=_('The email used for sign in to Bugzilla (BMO) :'))
```

Comment:
Also, the way this is coded, the form field is treated as required, but it shouldn't be. When you are testing the code you write you should check for conditions like this, e.g., test with an empty field, with an invalid email, with a valid email, etc.